packages/client/.github/contributing.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
```

Comment:
i don't think a `.github` subfolder in `packages/client` would do anything - can be moved up to root?